Makefile

```diff
@@ -277,3 +277,7 @@ test: .setup
 	fi
 	@cd $(BUILD_DIR) && $(CMAKE) && \
 		$(DEFINES) $(MAKE) --no-print-directory $(MAKEFLAGS) $(MAKECMDGOALS)
+
+xcode:
+	@cd $(BUILD_DIR) && $(CMAKE) -G Xcode && \
+		ln -s $(BUILD_DIR)/OSQUERY.xcodeproj $(SOURCE_DIR)/OSQUERY.xcodeproj
```

Comment:
Can you create this directly on the top-level directory?

Reply:
I tried but haven't found a proper cmake option…

Reply:
Basically, cmake generates project files in it's working directory, but also it generates a lot of other supporting stuff like `CMakeFiles` and a lot of `*.cmake`.
So simply running in `SOURCE_DIR` was not an option as it will get cluttered, but I haven't found an option to generate the project outside of working directory…